### PR TITLE
Update the locator bar and settings description

### LINKED ISSUE #5912: [FEATURE][locator] Add a goto locator (Request in QGIS)
### Request for documentation
From pull request QGIS/qgis#37977
Author: @nirvn
QGIS version: 3.16

**[FEATURE][locator] Add a goto locator**

## Description

Because I felt lame over the weekend showcasing QGIS and not being able to go to a longitude latitude coordinate, we have a brand new goto locator filter. This is co-presented by @Gustry who first had the idea and submitted a PR a while back. 

The goto indicator takes in the following strings:
- a pair of x and y (separated by a comma or a space)
- an openstreetmap/leaflet/openlayer formatted URL
- a google maps URL

The logic of the x y string is as follow:
- if the canvas CRS isn't WGS84:
  - if the coordinate fits within the canvas CRS bounds, add a canvas CRS result
  - if the coordinate fits within WGS84 bounds, add a WGS84 result (which will weight more than the above)
- if the canvas CRS is WGS84:
  - if the coordinate fits within the WGS84 bound, add a WGS84 result

The obligatory show off GIF:
Peek 2020-07-27 11-44

### Commits tagged with [need-docs] or [FEATURE]
"[FEATURE][locator] Add a goto locator"

### LINKED ISSUE #6094: [locator] autocomplete + allow restricting search to a single field in active layer filter (Request in QGIS)
### Request for documentation
From pull request QGIS/qgis#38633
Author: @3nids
QGIS version: 3.16

**[locator] autocomplete + allow restricting search to a single field in active layer filter**

### enable auto completion in locator
a locator filter can now return a completion list while preparing the search
the line edit will use the first matching completion and display it as light grey text
the completion can be triggered by pressing Tab key

### active layer locator filter field name restriction
the filter can now restrict the search to a specific field by typing @+field_name
auto-completion is provided for the field names

### demo
locator

(included in this PR the commit from #38628 )

### Commits tagged with [need-docs] or [FEATURE]
"[FEATURE] enable auto completion in locator\n\na locator filter can now return a completion list while preparing the search\nthe line edit will use the first matching completion and display it as light grey text\nthe completion can be triggered by pressing Tab key" "[FEATURE] active layer locator filter field name restriction\n\nthe filter can now restrict the search to a specific field by typing @+field_name\n\nauto-completion is provided for the field names"

### PR DESCRIPTION
**Pull request by DelazJ**
Fixes #6094, fixes #5912

### CODE REVIEW

File: docs/user_manual/introduction/qgis_gui.rst [hunk 1]
```diff
@@ -2512,14 +2512,52 @@ manage the map view.
...
-tag, keyword...) and you get a list that updates as you write.
-You can also limit the search scope using
-:ref:`locator filters <locator_options>`.
-Click the |search| button to select any of them and press the
-:guilabel:`Configure` entry for global settings.
+widget, helps you find and run any feature or options in QGIS:
+
+#. Click in the text widget to activate the locator search bar or press
+   :kbd:`Ctrl+K`.
+#. Type associated with the item you are looking for (name, tag, keyword...).
```
Comment (havatv) [inline docs/user_manual/introduction/qgis_gui.rst:2523]:
```suggestion
#. Type something associated with the item you are looking for
   (name, tag, keyword, ...).
```

File: docs/user_manual/introduction/qgis_gui.rst [hunk 2]
```diff
@@ -2512,14 +2512,52 @@ manage the map view.
...
+   By default, results are returned for the enabled locator filters but
```
Comment (havatv) [inline docs/user_manual/introduction/qgis_gui.rst:2524]:
```suggestion
   By default, results are returned for the enabled locator filters, but
```

File: docs/user_manual/introduction/qgis_gui.rst [hunk 3]
```diff
@@ -2512,14 +2512,52 @@ manage the map view.
...
+#. Click on a result to execute the corresponding action, depending on the type
+   of item.
+
+.. tip:: **Limit the lookup to one field of the active layer**
+
+  By default, a search with the "active layer features" filter (``af``) runs
+  through the whole attribute table of the layer. You can limit the search to
+  a particular field using the ``@`` prefix. E.g., ``af @name sal`` or
+  ``@name sal`` returns only the features whose "name" attribute contains 'sal'.
+  Text autocompletion is active when writing and suggestion can be applied
```
Comment (havatv) [inline docs/user_manual/introduction/qgis_gui.rst:2541]:
```suggestion
  Text autocompletion is active when writing and the suggestion can be applied
```

File: docs/user_manual/introduction/qgis_gui.rst [hunk 4]
```diff
@@ -2512,14 +2512,52 @@ manage the map view.
...
+  using :kbd:`Tab` key.
+
+Searching is handled using threads, so that results always become available as
+quickly as possible, regardless of whether any slow search filters may be
+installed. They also appear as soon as each result is encountered by each
```
Comment (havatv) [inline docs/user_manual/introduction/qgis_gui.rst:2546]:
```suggestion
quickly as possible, even if slow search filters are installed.
They also appear as soon as they are encountered by a
```

File: docs/user_manual/introduction/qgis_gui.rst [hunk 5]
```diff
@@ -2512,14 +2512,52 @@ manage the map view.
...
+filter, which means that e.g. a file search filter will show results one by one
+as the file tree is scanned. This ensures that the UI is always responsive even
```
Comment (havatv) [inline docs/user_manual/introduction/qgis_gui.rst:2548]:
```suggestion
as the file tree is scanned. This ensures that the UI is always responsive, even
```

File: docs/user_manual/introduction/qgis_gui.rst [hunk 6]
```diff
@@ -2602,6 +2633,19 @@ Click the icon to open the Plugin Manager dialog.
...
+Messaging
+---------
+
+The |messageLog| :sup:`Messages` button next to it opens the
+:guilabel:`Log Messages Panel` which has information on underlying
+processes (QGIS startup, plugins loading, processing tools...)
+
+Depending on the :ref:`Plugin Manager settings <setting_plugins>`,
+the status bar can sometimes show icons to the right to inform you
+about availability of |pluginNew| new or |pluginUpgrade| upgradeable
```
Comment (havatv) [inline docs/user_manual/introduction/qgis_gui.rst:2645]:
```suggestion
about the availability of new ( |pluginNew|) or  upgradeable (|pluginUpgrade|)
```

**Review by havatv — APPROVED** [review]
Looks fine. I have some suggestions.

**Comment by havatv** [comment]
Thanks @DelazJ.

**Comment by DelazJ** [comment]
No, there are 3.16 features and also no need to break translations for the first part of the PR. And more globally, with 3.16 around, we should definitely consider locking 3.10.

**Pull request merged**